prime-router/settings/STLTs/AL/al-phd.yml

```diff
@@ -0,0 +1,100 @@
+- name: "al-phd"
+  description: "Alabama Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "AL"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: [ ]
+  receivers:
+    - name: "full-elr"
+      organizationName: "al-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/AL/AL-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "ALNBS"
+        receivingApplicationOID: "2.16.840.1.114222.4.5.1"
```

Comment:
in order for batch headers to be set properly for this should be:
 receivingApplicationName" : "ALNBS^2.16.840.1.114222.4.5.1^ISO"
 receivingFacilityName: : "ALDOH^2.16.840.1.114222.4.1.3636^ISO"

Reply:
Perfect, that totally worked, thank you!